editor/fbx_importer_manager.cpp

```diff
@@ -0,0 +1,157 @@
+	} else {
+		List<String> args;
+		args.push_back("--version");
+		int exitcode;
+		Error err = OS::get_singleton()->execute(p_path, args, nullptr, &exitcode);
+
+		if (err == OK && exitcode == 0) {
+			success = true;
+		} else {
+			error = TTR("Error executing this file (wrong version or architecture).");
+		}
+	}
+
+	if (success) {
+		path_status->set_text(TTR("FBX2gLTF executable is valid."));
+		path_status->add_theme_color_override("font_color", path_status->get_theme_color(SNAME("success_color"), SNAME("Editor")));
+		get_ok_button()->set_disabled(false);
+	} else {
+		path_status->set_text(error);
+		path_status->add_theme_color_override("font_color", path_status->get_theme_color(SNAME("error_color"), SNAME("Editor")));
+		get_ok_button()->set_disabled(true);
+	}
+}
+
+void FBXImporterManager::_select_file(const String &p_path) {
+	fbx_path->set_text(p_path);
+	_validate_path(p_path);
+}
+
+void FBXImporterManager::_path_confirmed() {
+	String path = fbx_path->get_text();
+	EditorSettings::get_singleton()->set("filesystem/import/fbx/fbx2gltf_path", path);
+	EditorSettings::get_singleton()->save();
+}
+
+void FBXImporterManager::_browse_install() {
+	if (fbx_path->get_text() != String()) {
+		browse_dialog->set_current_file(fbx_path->get_text());
+	}
+
+	browse_dialog->popup_centered_ratio();
+}
+
+FBXImporterManager::FBXImporterManager() {
+	singleton = this;
+
+	set_title(TTR("Configure FBX Importer"));
+
+	VBoxContainer *vb = memnew(VBoxContainer);
+	vb->add_child(memnew(Label(TTR("FBX2gLTF2 is required for importing FBX files.\nPlease download it and provide a valid path to the binary:"))));
+	LinkButton *lb = memnew(LinkButton);
+	lb->set_text(TTR("Follow this link to obtain FBX2gLF2"));
+	lb->set_url("http://godotengine.org/fbx-import");
+	vb->add_child(lb);
+
+	HBoxContainer *hb = memnew(HBoxContainer);
+
+	fbx_path = memnew(LineEdit);
+	fbx_path->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	hb->add_child(fbx_path);
+	fbx_path_browse = memnew(Button);
+	hb->add_child(fbx_path_browse);
+	fbx_path_browse->set_text(TTR("Browse"));
+	fbx_path_browse->connect("pressed", callable_mp(this, &FBXImporterManager::_browse_install));
+	hb->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	hb->set_custom_minimum_size(Size2(400 * EDSCALE, 0));
+
+	vb->add_child(hb);
+
+	path_status = memnew(Label);
+	vb->add_child(path_status);
+
+	add_child(vb);
+
+	fbx_path->connect("text_changed", callable_mp(this, &FBXImporterManager::_validate_path));
+
+	get_ok_button()->set_text(TTR("Confirm Path"));
+	connect("confirmed", callable_mp(this, &FBXImporterManager::_path_confirmed));
+
+	browse_dialog = memnew(EditorFileDialog);
+	browse_dialog->set_access(EditorFileDialog::ACCESS_FILESYSTEM);
+	browse_dialog->set_file_mode(EditorFileDialog::FILE_MODE_OPEN_FILE);
+#if defined(OSX_ENABLED)
+#ifdef __x86_64__
+	browse_dialog->add_filter("FBX2glTF-darwin-x64");
+#else
+	browse_dialog->add_filter("FBX2glTF-darwin-arm64");
+#endif
```

Comment:
Maybe it's better to have it as universal binary?

Reply:
@bruvzg I have no idea to be honest, this will depend on @fire and @Calinou

Reply:
More details required.

1. Changing the cicd on gltf2fbx github repository.
2. Code signing changes
3. Documentation changes on the website

It probably should be done after this pr is merged.

Reply:
For now we're only building the x86_64 binary, so I changed it accordingly.

The FBX SDK seems to provide a universal binary for macOS so it should be possible to compile in theory, needs some tweaks to the FBX2glTF CI setup. PRs welcome :)